.gitmodules

```diff
@@ -38,3 +35,7 @@
 [submodule "safe-smart-account"]
 	path = safe-smart-account
 	url = https://github.com/safe-global/safe-smart-account.git
+[submodule "go-ethereum"]
+	path = go-ethereum
+	url = https://github.com/Tenderly/arb-go-ethereum.git
+	branch = live-tracing
```

Comment:
remove this change.
You should have a separate PR to our go-ethereum fork, point to it from your comment ("this pulls in + link"), and add a commit which sets go-ethereum to the specific commit of your geth-change.
We have merge checks that will not allow nitro commit to be merged before the geth PR is also merged.